specs/ExtendedProcessFailed.md

```diff
@@ -544,6 +545,9 @@ typedef enum COREWEBVIEW2_PROCESS_FAILED_REASON {
 
   /// The process died due to running out of memory.
   COREWEBVIEW2_PROCESS_FAILED_REASON_OUT_OF_MEMORY,
+
+  /// The process exited because its corresponding profile was deleted.
+  COREWEBVIEW2_PROCESS_FAILED_REASON_PROFILE_DELETED,
```

Comment:
Also need to update the MIDL3/WinRT definition of the enum below in the document.

Reply:
OK, I have added it.